dateparser/search/search.py

```diff
@@ -101,7 +103,7 @@ def parse_item(self, parser, item, translated_item, parsed, need_relative_base):
             parsed_item = parser.get_date_data(item)
         return parsed_item, is_relative
 
-    def parse_found_objects(self, parser, to_parse, original, translated, settings):
+    def parse_found_objects(self, parser, to_parse, original, translated, settings, language):
```

Comment:
```suggestion
    def parse_found_objects(self, parser, to_parse, original, translated, settings, language=None):
```

This is to avoid a breaking change (as this function is declared as public, even if it should be private).